src/instrument/plans/sim_plans.py

```diff
@@ -12,9 +14,8 @@
 
 from bluesky import plan_stubs as bps
 from bluesky import plans as bp
-
-from ..devices import sim_det
-from ..devices import sim_motor
+from ophyd.sim import motor as sim_motor  # noqa: F401
```

Comment:
move these two into file `../devices/ophyd_simulators.py`

"devices in `devices/`"

Reply:
done